src/Components/TrainingBravoPage.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { connect } from 'react-redux';
+
+import theme from '../styles/theme.style';
+import { completeTraining } from '../Store/Actions/programAction';
+import Button from './shared/Button';
+
+function getGoals(training) {
+  return Array.from(new Set(training.drills.reduce((acc, { goals }) => acc.concat(goals), [])));
+}
+
+export const TrainingBravoPage = props => {
+  const { navigation, route } = props;
+  const { training, program } = route.params;
+
+  const finishTraining = () => {
+    props.completeTraining({ training, program });
```

Comment:
This could also be called after rendering ? 🤔

Reply:
I think it would be better, in case the coach close the app after getting to this page.

Reply:
It's done